lib/sandboxed_renderer/init.js

```diff
@@ -103,16 +103,25 @@ function preloadRequire (module) {
   throw new Error('module not found')
 }
 
-if (window.location.protocol === 'chrome-devtools:') {
-  // Override some inspector APIs.
-  require('@electron/internal/renderer/inspector')
+switch (window.location.protocol) {
+  case 'chrome-devtools:': {
+    // Override some inspector APIs.
+    require('@electron/internal/renderer/inspector')
+    break
+  }
+  case 'chrome-extension:': {
+    // Inject the chrome.* APIs that chrome extensions require
```

Comment:
```suggestion
    // Inject the chrome.* APIs that devtools extensions require
```

Reply:
I think this is currently correct, although we don't advertise it as much you can actually use some normal Chrome extensions in Electron (not just devtools extensions)